recipes/ninja/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "NINJA_nj" %}
```

Comment:
Most recipes seem to use a dash instead of an underscore as separator, so I would suggest "ninja-nj".